python/ray/tests/test_advanced_3.py

```diff
@@ -683,6 +683,16 @@ def test_ray_address_environment_variable(ray_start_cluster):
     ray.shutdown()
 
 
+def test_ray_resources_environment_variable(ray_start_cluster):
+    address = ray_start_cluster.address
+
+    os.environ["RAY_RESOURCES"] = "{\"custom1\":1}"
+    ray.init(address=address, resources={"custom2": 1})
+
+    assert ("custom1" in ray.cluster_resources()
+            and "custom2" in ray.cluster_resources())
```

Comment:
Can you add a test that the env var has precedence?

Reply:
I thought we decided that env var does **not** have preference over cmd line flag. I'll add a conflict test case either way though.

Reply:
Let's make it override for now, and call the env var name OVERRIDE.